walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -22,44 +22,59 @@
 import java.util.List;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    Long duration = request.getDuration();
-    Iterator<Event> iterator = events.iterator();
-    int start = 0;
-    Collection<TimeRange> timeBlock = new ArrayList();
-    if(duration>TimeRange.WHOLE_DAY.duration()){
-        return Arrays.asList();
-    }
-    if(events.size()==0 ){
-        return Arrays.asList(TimeRange.WHOLE_DAY);
-    }
-    /**returns all time intervals between events*/
-    while (iterator.hasNext()) {
-        Event e = iterator.next();
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        Long duration = request.getDuration();
+        int start = 0;
+        Collection<TimeRange> timeBlock = new ArrayList();
+        if(duration>TimeRange.WHOLE_DAY.duration()){
+            return Arrays.asList();
+        }
+        if(events.size()==0 ){
```

Comment:
Spacing `if (...) {`. This comment follows for all other if/while/for/else if statements below.